src/core/server/saved_objects/types.ts

```diff
@@ -35,66 +35,17 @@ export {
 import { LegacyConfig } from '../legacy';
 import { SavedObjectUnsanitizedDoc } from './serialization';
 import { SavedObjectsMigrationLogger } from './migrations/core/migration_logger';
+import { SavedObject } from '../../types';
```

Comment:
Would you mind updating https://github.com/elastic/kibana/blob/ec2134d221aadc732c30a7bc88953cebcdf23c6f/src/core/public/saved_objects/index.ts#L35-L39
to export types from  `../../types` instead of `server/saved_objects/type`?